dll/cpl/sysdm/licence.c

```diff
@@ -23,16 +24,23 @@ OnInitDialog(HWND hDlg, PLIC_CONTEXT pLicInfo)
     PCSTR LicenseText;
 
     pLicInfo->hIcon = LoadImage(hApplet,
-                                MAKEINTRESOURCE(IDI_CPLSYSTEM),
+                                MAKEINTRESOURCE(IDI_LICENSE),
                                 IMAGE_ICON,
-                                16,
-                                16,
-                                0);
+                                GetSystemMetrics(SM_CXICON),
+                                GetSystemMetrics(SM_CYICON),
+                                LR_DEFAULTCOLOR);
+    pLicInfo->hIconSm = LoadImage(hApplet,
```

Comment:
How does it look like, if only hIcon and in the SendMessage call below, only ICON_BIG is set?

Reply:
![VirtualBox_ReactOS_DBG_13_10_2022_21_16_17](https://user-images.githubusercontent.com/86486978/195622059-2d311089-5395-4bc6-b3d1-d4b786d75226.png)

The icon is scaled so setting icon manually looks better.